### ISSUE TITLE
mpv fails to detect BD structure

### ISSUE THREAD

**Issue by fhlfibh** [mpv-player/mpv#4403]
Hi,

### mpv version and platform
mpv 0.25.0-31-g95d5fe1 (C) 2000-2017 mpv/MPlayer/mplayer2 projects
 built on Thu May  4 20:23:42 MSK 2017
ffmpeg library versions:
   libavutil       55.61.100
   libavcodec      57.92.100
   libavformat     57.72.101
   libswscale      4.7.101
   libavfilter     6.84.101
   libswresample   2.8.100
ffmpeg version: N-85548-g3390a2b

Normally, when I start as `mpv /bddir` where `bddir` contains a _BD structure_ (i.e. a dir with BDMV et al.), mpv somehow recognizes it as such, picks the biggest file in STREAM dir (afaik), and starts playback. I even don't have to specify `bd:// --bluray-device=` or anything.

But recently I came across a _BD structure_ that doesn't work like above: seems mpv fails to detect it as such, but starts to load playlists, tries to play image files, etc instead of "normal" playback.
In other words, I can start the actual movie only if I specify the correct .m2ts file in STREAM manually.
Adding `bd:// --bluray-device=/bddir/` doesn't help. I suspect, the problem is that weird _BD structure_ contains hevc videos, not h264. However, I'm not sure.

The problem is not that serious, but rather annoying...

### Log file
Log excerpts (full logs are quite big, but I can upload them too if necessary)
Correct playback http://sprunge.us/XONi
Failed playback http://sprunge.us/WGeY

### Sample files
It's a 50Gb _BD structure_ of 800+ files...

**Comment by wm4** [comment]
Log with --bluray-device? Probably not our bug anyway. Try with libbluray git master or so.

**Comment by wm4** [comment]
so I suspect libbluray doesn't handle them yet.

**Comment by fhlfibh** [comment]
I sent a report to libbluray-devel mailing list. Not sure if it can help, though.
